recipe/meta.yaml

```diff
@@ -1,35 +1,36 @@
+{% set name = "twine" %}
 {% set version = "3.7.1" %}
 
 package:
-  name: twine
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/t/twine/twine-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 28460a3db6b4532bde6a5db6755cf2dce6c5020bada8a641bb2c5c7a9b1f35b8
 
 build:
-  noarch: python
```

Comment:
Do not remove noarch python if we do not update the version

Reply:
ok